hexa/pipelines/management/commands/pipelines_runner.py

```diff
@@ -236,7 +244,10 @@ def run_pipeline_kube(run: PipelineRun, image: str, env_vars: dict):
 def run_pipeline_docker(run: PipelineRun, image: str, env_vars: dict):
     from subprocess import PIPE, STDOUT, Popen
 
-    docker_cmd = f'docker run --privileged -e HEXA_ENVIRONMENT=CLOUD_PIPELINE -e HEXA_RUN_ID={env_vars["HEXA_RUN_ID"]} -e HEXA_SERVER_URL={env_vars["HEXA_SERVER_URL"]} -e HEXA_TOKEN={env_vars["HEXA_TOKEN"]} -e HEXA_WORKSPACE={env_vars["HEXA_WORKSPACE"]} --network openhexa --platform linux/amd64 --rm {image} pipeline cloudrun'
+    env_vars.pop("HEXA_PIPELINE_NAME", "")
```

Comment:
Why is it needed ?

Reply:
I've seen that we really don't use it for the run_docker, but I can keep it (for future usage ?)